### ISSUE TITLE
Move recipient address of projects to purple address table ( write migration needed for existing projects)

### ISSUE THREAD

**Issue by mohammadranjbarz** [Giveth/giveth-dapps-v2#715]

**Comment by divine-comedian** [comment]
is there a way for users to test this?

**Comment by MoeNick** [comment]
@mohammadranjbarz pls provide test scenario
@jeevanplus007 pls have it in your test sheet.

**Comment by jeevanplus007** [comment]
@mohammadranjbarz  Any Test Scenarios or Snapshots you can provide for reference. ?

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Any Test Scenarios or Snapshots you can provide for reference. ?

Every wallet address of any project should be here
https://serve.giveth.io/admin/resources/ProjectAddress

and For tests, you can create a new project and then you should check admin panel to see whether that address appeared in admin panel or not

**Comment by divine-comedian** [comment]
testing on `https://serve.giveth.io/admin/resources/ProjectAddress`

I created a project "purple list migration" and I look up the recipient address in the project address page - I can see two results for the address, one for mainnet, one for gnosis chain

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/67759413/182403035-b7c78564-d6d3-4895-8144-9663ac2fcb61.png">


A few confusing things about this page however - The title is empty for most addresses in the list, but the project title seems to appear under the `project id` column. There's also an Id. column that I assume is the actual project Id.

Should we move the info in project Id to title and then remove the project Id column altogether? This would make search for a project address by title much more intuitive.

**Comment by mohammadranjbarz** [comment]
> testing on `https://serve.giveth.io/admin/resources/ProjectAddress`
> 
> I created a project "purple list migration" and I look up the recipient address in the project address page - I can see two results for the address, one for mainnet, one for gnosis chain
> 
> <img alt="image" width="1440" src="https://user-images.githubusercontent.com/67759413/182403035-b7c78564-d6d3-4895-8144-9663ac2fcb61.png">
> 
> @mohammadranjbarz A few confusing things about this page however - The title is empty for most addresses in the list, but the project title seems to appear under the `project id` column. There's also an Id. column that I assume is the actual project Id.
> 
> Should we move the info in project Id to title and then remove the project Id column altogether? This would make search for a project address by title much more intuitive.

You know we just have a relation between projectAddress and project, so admin bro automatically show project title instead of projected ( I think it's good because we don't have any sense about projected, but project title is good and human friendly)
about title, it's empty for all recipient addresses but I added this field because I thought maybe when admin add an address ( a related address or purple list not recipient) maybe it's good to have a title, for instance you can put a title **charity address of project** or something like that.

TL;DR
* I can't change `projectId` display in amdin bro ( at least cant change it easily)
* I can delete **title** from admin bro columns or even from entity, but I think it would be good to keep it

**Comment by divine-comedian** [comment]
@mohammadranjbarz 

Okay, so most migrated projects will automatically have their project title (project Id) inside of this admin view

when we add in additional addresses associated with a project via the verification form:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/67759413/182676221-9ac9544a-da96-4878-b294-7db457806c2c.png">

are those addresses automatically added in this project address table as well? do they have the same project Id associated with them?

I can't think of a specific use case of title unless you mean it as a section for an admin to leave a note for other admins. Are we able to add a notes column and remove title? Is it possible to move the Project Id column where the title column is?
